src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -417,7 +418,12 @@ public Response postProgressReport(
       @FormDataParam("ethicsApprovalRequiredFile") FormDataContentDisposition ethicsFileDetails) {
     User user = userService.findUserByEmail(authUser.getEmail());
     DataAccessRequest parentDar = dataAccessRequestService.findByReferenceId(parentReferenceId);
-    checkAuthorizedUpdateUser(user, parentDar);
+    try {
+      checkAuthorizedUpdateUser(user, parentDar);
+    } catch (Exception e) {
```

Comment:
Can we be more specific than just `Exception`?

Reply:
We can, but we should be really wrapping the larger method in this pattern (like the other methods in this class). Since it isn't currently used, I punted on the bigger refactor for now and am catching the generic Exception as my tell.

Reply:
If you look at `createExceptionResponse(e)`, we try to be smart about translating various exceptions into logical HTTP response codes. This is a consistent pattern we use to trap unexpected exceptions from being returned as unclear responses.